### PR TITLE
Pbw 3165

### PR DESCRIPTION
**Pull request by chibzu**

### CODE REVIEW

File: scss/mixins/_mixins.scss [hunk 1]
```diff
@@ -221,4 +221,13 @@
...
 }
 
-// add font-smoothing
\ No newline at end of file
+// add font-smoothing
+
+
+// Vendor prefixing for shorthand properties
+@mixin vendor-prefixes($property, $value) {
+        -webkit-#{$property}: $value;
```
Comment (mistermendez) [inline scss/mixins/_mixins.scss]: hotness :fire:

File: tests/components/scrubberBar-test.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+        controlBarWidth={500}
+        seeking={false}
+        controller={mockController}
+        currentPlayhead={30}
+        duration={60}
+        />
+    );
+
+    var playhead = TestUtils.findRenderedDOMComponentWithClass(DOM, 'playhead').getDOMNode();
+    expect(playhead.style.visibility).toBe('hidden');
```
Comment (mistermendez) [inline tests/components/scrubberBar-test.js]: FAIL  tests/components/scrubberBar-test.js (0.394s)
● ScrubberBar › it enters ad mode
  - Expected: '' toBe: 'hidden'
        at Spec.<anonymous> (/Users/jmendez/repos/html5-skin/tests/components/scrubberBar-test.js:144:39)

**Comment by mistermendez** [comment]
other than partial unit test fail... LGTM

**Comment by chibzu** [comment]
Good catch, thanks. Updated unit test for the CSS approach.

**Pull request merged**